### PR TITLE
Refactored openebs install test case and refactored clean-up tasks .

### PR DESCRIPTION
**Pull request by amitbhatt818**
- Refactored openebs install job and added blocks into the test case.
- Removed app ns from clean-up task

### CODE REVIEW

File: litmus/director/tcid-iuoi02-openebs-install/test.yml [hunk 1]
```diff
@@ -95,34 +95,11 @@
...
-        ## Fetch node-2 details of the cluster
-        - name: Fetch node-2 details of the cluster
-          set_fact:
-            node2_id: "{{ node_cluster.json.data[1].id}}"
 
         ## Labeling the node-1 of connected Cluster with controlPlaneNode=true and dataPlaneNode=true
         - name: Giving POST request on labelnodes
           uri:
-            url: "{{ director_url }}/v3/groups/{{ group_id }}/nodes/{{ node1_id }}/?action=labelnodes"
+            url: "{{ director_url }}/v3/groups/{{ group_id }}/nodes/{{ node_id[0] }}/?action=labelnodes"
```
Comment (harshshekhar15) [inline litmus/director/tcid-iuoi02-openebs-install/test.yml]: How are we getting the `node_id`?

File: litmus/director/tcid-iuoi02-openebs-install/test.yml [hunk 2]
```diff
@@ -144,79 +121,127 @@
...
-            status_code: 201
-          register: install_openebs
+            status_code: 200
+          register: openebses_active
 
-        ## Fetching installation manifest
-        - name: Get manifest for installing openebs
-          set_fact:
-            installopenebs: "{{ install_openebs.json.installationManifest }}"
+        - block:
```
[the remark this answers is not in the document]
Comment (amitbhatt818) [reply litmus/director/tcid-iuoi02-openebs-install/test.yml]: Yes we are now using multiple blocks

**Comment by amitbhatt818** [comment]
https://github.com/mayadata-io/oep-e2e/blob/43e765d375266ee96fdca2a8d110b328c7215cc4/litmus/director/tcid-iuoi02-openebs-install/test.yml#L71
From this block we are storing node_id

File: litmus/director/tcid-iuoi02-openebs-install/test.yml (hunk 1 above)
Comment (amitbhatt818) [reply litmus/director/tcid-iuoi02-openebs-install/test.yml]: https://github.com/mayadata-io/oep-e2e/blob/43e765d375266ee96fdca2a8d110b328c7215cc4/litmus/director/tcid-iuoi02-openebs-install/test.yml#L71
From this block we are storing node_id

**Pull request merged**